src/main/java/Ixo.java

```diff
@@ -0,0 +1,151 @@
+import java.util.Scanner;
+
+
+public class Ixo {
+
+    public final static String SEPARATOR = "____________________________________________________";
+
+    public static void menu() {
+        System.out.println("1. Task List");
+    }
```

Comment:
Function names should ideally be verbs. So a name like `showMenu` or `printMenu` would be more apt.